src/screens/OtherProfile/index.js

```diff
@@ -642,9 +642,12 @@ const OtherProfile = () => {
           extraData={[feeds]}
           data={handleDataFeed()}
           onScroll={handleScroll}
-          onEndReach={__handleOnEndReached}
           onRefresh={handleRefresh}
           refreshing={loading}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been replaced with `onMomentumScrollEnd`, which now calls both `onWillSendViewPostTime` and `fetchNextFeeds`. This is a significant change in how the feed loading behavior works. It's important to test this thoroughly to ensure that it doesn't cause performance issues or unexpected behavior, especially when scrolling quickly or when the network connection is slow.

```diff
-  onEndReach={__handleOnEndReached}
+  onMomentumScrollEnd={(event) => {
+    onWillSendViewPostTime(event, feeds);
+    fetchNextFeeds(event);
+  }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->